src/header.ts

```diff
@@ -4,14 +4,126 @@
 
+/**
+ * header html template declared inline so we can use i18next
+ */
+const getTemplate = () => {
+  return (`<nav class="navbar">
+  <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#headerNavBar">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#" data-header="appLink"></a>
+  </div>
+  <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav" data-header="mainMenu">
+
+      </ul>
+      <ul class="nav navbar-nav navbar-right" data-header="rightMenu">
+          <li class="hidden" data-header="optionsLink">
+              <a href="#" data-toggle="modal" data-target="#headerOptionsDialog" title="${i18n.t('phovea:ui.options')}">
+                  <i class="fa fa-cog fa-fw" aria-hidden="true"></i>
+                  <span class="sr-only">${i18n.t('phovea:ui.openOptionsDialog')}</span>
+              </a>
+          </li>
+          <li class="hidden" data-header="aboutLink">
+              <a href="#" data-toggle="modal" data-target="#headerAboutDialog" title="${i18n.t('phovea:ui.about')}">
+                  <i class="fa fa-info fa-fw" aria-hidden="true"></i>
+                  <span class="sr-only">${i18n.t('phovea:ui.openAboutDialog')}</span>
+              </a>
+          </li>
+          <li class="hidden" data-header="bugLink">
+              <a href="#" data-toggle="modal" data-target="#headerReportBugDialog" title="${i18n.t('phovea:ui.reportBug')}">
+                  <i class="fa fa-bug fa-fw" aria-hidden="true"></i>
+                  <span class="sr-only">${i18n.t('phovea:ui.reportBug')}</span>
+              </a>
+          </li>
+          <li class="hidden" data-header="helpLink">
+              <a href="//caleydo.org" target="_blank" title="${i18n.t('phovea:ui.openHelpPage')}">
+                  <i class="fa fa-question fa-fw" aria-hidden="true"></i>
+                  <span class="sr-only">${i18n.t('phovea:ui.openHelpPage')}</span>
+              </a>
+          </li>
+      </ul>
+  </div>
+</nav>
+
+<!-- About Dialog -->
+<div class="modal fade" id="headerAboutDialog" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+      <div class="modal-content">
+          <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="${i18n.t('phovea:ui.close')}">
+                  <span aria-hidden="true">&times;</span>
+              </button>
+              <h4 class="modal-title">${i18n.t('phovea:ui.about')}</h4>
+          </div>
+          <div class="modal-body" data-header="about">
+              <div class="metaData">${i18n.t('phovea:ui.loading')}</div>
+              <div class="caleydoInfo">
+                  <a class="logo" href="https://phovea.caleydo.org" target="_blank"><img
+                          src="assets/caleydo_c.svg"></a>
```

Comment:
Since you moved the template from an HTML file to this function the asset file `caleydo_c.svg` is not included in the webpack bundle and results in a 404 error.

![grafik](https://user-images.githubusercontent.com/5851088/70255596-5efec480-1787-11ea-8fbe-412a869bb8a2.png)

I assume that you can fix it by adding the following import to the top of the file:

```ts
import * as caleydoLogo from 'url-loader!./assets/caleydo_c.svg';
```

And then using the resource here:

```suggestion
                          src="${caleydoLogo}"></a>
```